tests/org.jboss.tools.vpe.preview.editor.test/src/org/jboss/tools/vpe/preview/editor/test/editor/RefreshOptionsTest.java

```diff
@@ -130,10 +130,13 @@ private void saveEditor() throws Exception{
 	}
 	
 	private void setRefreshPreferences(boolean onSave, boolean onChange){
-		IPreferenceStore preferences = Activator.getDefault().getPreferenceStore();
-		preferences.setValue(REFRESH_ON_SAVE_PREFERENCES, onSave);
-		preferences.setValue(REFRESH_ON_CHANGE_PREFERENCES, onChange);
-		visualEditor.getActionBar().updateRefreshItemsAccordingToPreferences();
```

Comment:
i believe this caused RefreshOptionsTest instability - changing onSave/onChange via preferences is not enough because there's no listener for these preferneces to actually reflect the changes